### ISSUE TITLE
[Screen Reader-Control Playground-Diffeditor (editor)] Screen reader does not announce tooltip information while press alt+f1 key to expand accessibility option when lands on "old content to change" editor field.[Bug] 

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/monaco-editor#2859]
### Actual:

Screen reader not announce tooltip information while press alt+f1 key to expand accessibility option when lands on "old content to change" editor field.

**Observation:**
When press alt+f1 to expand tooltip information, Narrator don't announce any information (Blank).
Issue appear only with narrator
NVDA and JAWS announce dialogue information as "Line 1, Column 1 in a read-only pane of a diff editor" etc.

### Expected:
If any tooltip appear on the blade it should be announce by screen reader as well.

**GitHubTags:**#A11yMAS;#A11yTCS;#A11ySev2;#Desktopweb;#WCAG1.3.1;#Win11;#AILimited;#FTP;#Visual Studio Code Client;#VisualStudioCode-Web-Apr2021;#a11y-monacoeditor;#Narrator;#NVDA;

### Environment Details:
#URL: https://aka.ms/portalfx/playground
Chromium edge: 98.0.1100.3 (Official build) dev (64-bit)
Chrome: Version 96.0.4664.110 (Official Build) (64-bit)

### Repro Steps:

1. Open https://aka.ms/portalfx/playground and valid credentials.
2. Navigate to the controls and select it.
3. Navigate to the "Diff Editor (Editor)" an select it.
4. Navigate to the playground option.
5. Navigate to the control and control properties appearing on the "Sample" Pane.(Diff Editor (Editor))  
6. Navigate "Old content to change" editor field.
7. Press alt+f1 to expand the tooltip
8. Verify the issue.

### User Impact:
Screen reader user who rely on screen reader will unaware about that tooltip information if while pressing alt+f1 tooltip appear but not announce any information by screen reader.

### Attachment:
![2859_Screen reader does not announce tooltip information while press alt plus f1 key to expand accessibility option when lands on old content to change editor field](https://user-images.githubusercontent.com/60880975/147544588-a9b3f3ef-f77c-4311-ad63-51008bb3dace.gif)
![2859_MAS1 3 1-Controlplayground-Diffeditor-not announce tooltip info](https://user-images.githubusercontent.com/60880975/147544727-7074b739-0283-4c77-bac8-552a831c72ac.png)

**Comment by isidorn** [comment]
Please file against narator. We primarily support nvda and jaws.

Closing as we have no plans to tackle this. There are bigger problems with Narator (not very web aria compatible)
